docs/release-notes/sso-auto-redirect.rst

```diff
@@ -0,0 +1,12 @@
+:orphan:
+
+** New Features**
+
+-  Master Configuration: Add `always_redirect` option in OIDC and SAML configurations. When enabled, this option
+      bypasses the standard Determined login screen and directly routes users to the configured SSO
+      provider. This redirection persists unless the user explicitly signs out within the WebUI.
+
+**Improvements**
+
+-  WebUI: Redirect SSO users to SSO provider authentication URIs when the provided session token is expired,
+      rather than showing the determined Sign-In page.
```

Comment:
:orphan:

** New Features**

-  Master Configuration: Add an ``always_redirect`` option to OIDC and SAML configurations. When enabled, this option
      bypasses the standard Determined sign-in screen and directly routes users directly to the configured SSO
      provider. This redirection remains active unless the user explicitly signs out within the WebUI.

**Improvements**

-  WebUI: Redirect SSO users to the SSO provider's authentication URIs when their session token has expired, instead of displaying the Determined sign-In page.